README.md

```diff
@@ -341,6 +341,7 @@ nc.QueueSubscribe("foo", "job_workers", func(_ *Msg) {
 // the connection in reconnecting state if it failed to connect right away.
 nc, err := nats.Connect(nats.DefaultURL,
     nats.RetryOnFailedConnect(true),
+    nats.PingInterval(time.Second),
```

Comment:
1 sec for the ping interval is super low, 30s to 2 minutes would be a better duration. Though you should not have to set this so would be good to double check the root cause at what what is happening here

Reply:
I can bump it up to 30 I agree with you.